assets/stylesheets/_layout.scss

```diff
@@ -0,0 +1,26 @@
+// ----- Flex-box mixin (work in progress)
+@mixin grid($flex, $direction:column) {
+  flex-direction: $direction;
+  display:flex;
+  
+  & > li {      
+	  flex:$flex;
+    list-style:none;
+    margin:0 1em 0 0;
+    padding: 1em;
+  }
+  
+  @media all and (min-width: 480px) {
+  & {flex-direction: row;}  
+  }
+}
+
+// ----- Main Layout styles
+main{
+	max-width: 60em;
+	margin: 1em auto 0;
+}
+
+.content {
+	margin:0 auto;
+}
```

Comment:
Please update your Sublime text preferences with what's listed in the [code style guide](https://github.com/GothamSass/chrysler/wiki/Code-style-guidelines) and then re-save this file.